dev-requirements.txt

```diff
@@ -0,0 +1,4 @@
+codecov==2.1.12
+flake8==3.9.2
```

Comment:
should we bump flake8 to 4.0.1 at the same time?

Reply:
Fixed